lib/internal/checkUrl.js

```diff
@@ -66,6 +85,15 @@ function checkUrl(link, baseUrl, options, callback)
 		}
 		
 		linkObj.clean(link);
+
+		if ( cache !== undefined && cache !== null )
```

Comment:
Actually... damn... having thought about this more, this will vary from engine to engine. We should go with what you had, but please change the formatting to:
```js
if (cache!==undefined && cache!==null)
```